packages/adapter/lib/adapter/adapter.js

```diff
@@ -19,26 +19,26 @@
 //   noNamespace:           return short names of objects and states in objectChange and in stateChange
 //   strictObjectChecks:    flag which defaults to true - if true, adapter warns if states are set without an corresponding existing object
 
-const net               = require('net');
-const fs                = require('fs-extra');
-const extend            = require('node.extend');
-const util              = require('util');
-const os                = require('os');
-const EventEmitter      = require('events').EventEmitter;
-const tools             = require('./tools');
-const pidUsage          = require('pidusage');
-const deepClone         = require('deep-clone');
-const EXIT_CODES        = require('./exitCodes');
-const {PluginHandler}   = require('@iobroker/plugin-base');
-const controllerVersion = require('../package.json').version;
-
-const password          = require('./password');
-
-const { FORBIDDEN_CHARS } = tools;
-const DEFAULT_SECRET    = 'Zgfr56gFe87jJOM';
+const net = require('net');
+const fs = require('fs-extra');
+const extend = require('node.extend');
+const util = require('util');
+const os = require('os');
+const EventEmitter = require('events').EventEmitter;
+const {tools} = require('@iobroker/js-controller-common');
+const pidUsage = require('pidusage');
+const deepClone = require('deep-clone');
+const {EXIT_CODES} = require('@iobroker/js-controller-common');
+const {PluginHandler} = require('@iobroker/plugin-base');
+const controllerVersion = require(tools.getControllerDir() + '/package.json').version;
```

Comment:
Ich denke wir sollten aus `js-controller` die package.json-Version als property exportieren, anstatt wieder mit Pfaden zu jonglieren.

Reply:
Für mich ok, wobei durch getControllerDir schon mal schöner als vorher

Reply:
Im Endeffekt aber eigentlich auch egal. Mit `lerna` fixed versioning haben alle Pakete die gleiche Version, wenn wir bei `exact: true` bleiben (was wir sollten).

Reply:
Denke bleiben bei exact: true ... also nimm vom lokalen paket :-)

Reply:
mabe add a comment that we remember why we did it then the "local" way